charts/pgbouncer/templates/pgbouncer-backend-secret.yaml

```diff
@@ -0,0 +1,21 @@
+################################
```

Comment:
we can remove it, still don't have any final requirements, so i do assumption that customer can configure everything via ENV variables.